src/App/routes.tsx

```diff
@@ -317,6 +317,62 @@ const surge = customWrapRoute({
     },
 });
 
+const surgeOverview = customWrapRoute({
+    path: 'overview',
+    component: {
+        render: () => import('#views/SurgeOverview'),
+        props: {},
+    },
+    parent: surge,
+    context: {
+        title: 'Surge Overview',
+        visibility: 'anything',
+    },
+});
+
+const operationalToolbox = customWrapRoute({
+    path: 'operational-toolbox',
+    component: {
+        render: () => import('#views/OperationalToolbox'),
```

Comment:
Lets add surge prefix
```suggestion
        render: () => import('#views/SurgeOperationalToolbox'),
```